integration_test/dapp_tests/uniswap/uniswapTest.js

```diff
@@ -0,0 +1,253 @@
+const hre = require("hardhat"); // Require Hardhat Runtime Environment
```

Comment:
it might be worth updating this to use typescript, but we can do that in a separate pr 

for now this is fine

Reply:
Good call, can look into that after we're done with the test suite